.github/workflows/remove-deploy-previews.yml

```
@@ -1,299 +1,299 @@
-        with:
-          credentials_json: "${{ secrets.GCP_SA_CREDENTIALS }}"
-
-      - name: Setup Cloud SDK
-        uses: "google-github-actions/setup-gcloud@v1.1.1"
-
-      - name: Delete PR deploy preview
-        run: |-
-          gcloud run services delete ${{ needs.branch-name.outputs.lowercase }}-website-preview \
-            --region=${{ secrets.REGION }} \
-            --quiet            
+# name: remove-deploy-previews
```

actionlint (1.7.12)

[error] workflow is empty

(syntax-check)